plugins/node/opentelemetry-instrumentation-mongodb/src/types.ts

```diff
@@ -36,6 +44,11 @@ export interface MongoDBInstrumentationConfig extends InstrumentationConfig {
    * @default undefined
    */
   responseHook?: MongoDBInstrumentationExecutionResponseHook;
+
+  /**
+   * Custom serializer function for the db.statement tag
+   */
+  dbStatementSerializer?: DbStatementSerializer;
```

Comment:
Please also add it to README under "Mongo instrumentation Options"

Reply:
done